### PR TITLE
Dockerfile.client

### PR DESCRIPTION
**Pull request by cmullenmm**
Added docker file to build the client code. Also added requirements.txt to track python libraries which need to be installed. This can then be run using pip install -r requirements.txt. Added a Makefile for build process

Using the docker build -f flag which now supports multiple Dockerfiles in the same repository for cases like this.

### CODE REVIEW

File: src/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+if [ -z "$1" ]; then
+
+  if [ -z $QASINO_HOST ]; then
+    echo "Please spcify a QASINO_HOST."
```
Comment (bradw456) [inline src/docker-entrypoint.sh]: Typo here.  spcify -> specify

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+VERSION := `head -1 debian/changelog | awk '{print $2}' | grep -o '\d\+.\d\+'`
```
Comment (bradw456) [inline Makefile]: My environment had an issue with this.  It did not get the version number.  Perhaps my version of grep?  This worked for me though (and supports versions with more then one dot):

      head -1 debian/changelog | awk '{print $2}' | grep -o -e '\([0-9\.]\+\)' | tr -d '()'

File: src/docker-entrypoint.sh (hunk 1 above)
Comment (cmullenmm) [inline src/docker-entrypoint.sh]: Fixed in the latest commit

File: Makefile (hunk 2 above)
Comment (cmullenmm) [inline Makefile]: Ah yes. The version of grep does matter. I have updated it to your command as it is more robust

**Pull request merged**